framework/web/UrlManager.php

```diff
@@ -143,11 +143,6 @@ class UrlManager extends Component
      */
     public $normalizer = false;
 
-    /**
-     * @var string the cache key for cached rules
-     * @since 2.0.8
-     */
-    protected $cacheKey = __CLASS__;
```

Comment:
The property should become private. Getter should return default value as you've proposed if property it's not set explicitly.